src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
This is correct nd valid code, however you can make it better and cleaner in two steps:

1. replace :first-child with :nth-child(...guess what you should put her...)
2. utilize https://sass-lang.com/documentation/at-rules/control/for/
3.